src/nat/eval/evaluate.py

```
@@ -169,62 +170,72 @@ async def run_one(item: EvalInputItem):
+
+                    runner_result = None
+                    intermediate_future = None
+
+                    try:
+                        # Start usage stats and intermediate steps collection in parallel
+                        intermediate_future = pull_intermediate()
+                        runner_result = runner.result()
+                        base_output = await runner_result
+                        intermediate_steps = await intermediate_future
+                    except NotImplementedError as e:
+                        logger.error("Failed to run the workflow: %s", e)
```

Ruff (0.14.8)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)